akka-docs/rst/java/code/docs/stream/StreamTestKitDocTest.java

```diff
@@ -120,11 +120,11 @@ public void sinkActorRef() throws Exception {
     final TestProbe probe = new TestProbe(system);
     final Cancellable cancellable = sourceUnderTest
       .to(Sink.actorRef(probe.ref(), Tick.COMPLETED)).run(mat);
-    probe.expectMsg(Duration.create(1, TimeUnit.SECONDS), Tick.TOCK);
+    probe.expectMsg(Duration.create(3, TimeUnit.SECONDS), Tick.TOCK);
     probe.expectNoMsg(Duration.create(100, TimeUnit.MILLISECONDS));
```

Comment:
I think the doc snippet tried to demonstrate a periodic Tick here and assert that there is at least one period between the ticks. This is racy no matter what, maybe this sample needs to be rewritten using something else.